ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONServer.java

```diff
@@ -53,8 +54,8 @@ public JSONServer(ServerCoreProfile coreProfile) {
         featureRepository.addFeatureProfile(coreProfile);
     }
 
-    public void enableWSS(SSLContext sslContext) {
-        listener.enableWSS(sslContext);
+    public void enableWSS(WssFactoryBuilder wssFactoryBuilder) {
```

Comment:
The client returned an object of itself, this one doesn't.

Reply:
Fixed.